mqt/qcec/bindings.cpp

```diff
@@ -96,11 +96,15 @@ static std::unique_ptr<EquivalenceCheckingManager> createManagerFromOptions(
     // Simulation
     const double fidelityThreshold = 1e-8,
 
-    const std::size_t maxSims   = std::max(16U,
-                                           std::thread::hardware_concurrency() -
-                                               2U),
-    const StateType&  stateType = StateType::ComputationalBasis,
+    const std::size_t maxSims = std::max(16U,
+                                         std::thread::hardware_concurrency() -
+                                             2U),
+    // Parameterized
+    const double      parameterizedTol          = 1e-12,
+    const std::size_t nAdditionalInstantiations = 0,
```

Comment:
Are these options purposely sandwiched between the "Simulation" options?

Reply:
Put it afterwards now